cookbooks/travis_build_environment/templates/default/etc/apt/confold.erb

```diff
@@ -0,0 +1,7 @@
+// Managed by chef on <%= node.name %> :heart_eyes_cat:
+// vim:filetype=aptconf
+
+Dpkg::Options {
+  "--force-confdef";
+  "--force-confold";
+}
```

Comment:
for anyone else who wanted to look it up to see exactly what it does:

              confold:  If a conffile has been modified and the version in the
              package  did  change,  always  keep  the  old  version   without
              prompting,  unless  the  --force-confdef  is  also specified, in
              which case the default action is preferred.

              confdef: If a conffile has been modified and the version in  the
              package  did  change,  always  choose the default action without
              prompting. If there is no default action it will stop to ask the
              user  unless  --force-confnew  or  --force-confold  is also been
              given, in which case it  will  use  that  to  decide  the  final
              action.